cpp/src/aztec/honk/sumcheck/polynomials/univariate.hpp

```diff
@@ -309,4 +295,67 @@ template <class Fr, size_t view_length> class UnivariateView {
         return os;
     }
 };
+
```

Comment:
These methods don't seem to be used anywhere. You see some use for them in the future?

Reply:
The main use case is to convert an array of `Univariate` into `UnivariateView`. The main use case would be to let Sumcheck decide the required degree of the relation evaluation, rather than hardcoding it inside the relation. The `_aux` version could also be used to create an array of only the polynomials required by the relation, and it could help us implement the optimization where we extend each edge only up to the maximum degree that is required over all relations (for example, `L_LAST` only needs degree 3). 

I'll save this comment as the context might help.